docs/source/accelerators/gpu_intermediate.rst

```diff
@@ -104,6 +103,12 @@ In these situations you should use `dp` or `ddp_spawn` instead.
 
 Distributed Data Parallel 2
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. warning::
+    The DDP2 strategy is no longer supported. For single-node use, we recommend ``strategy='ddp'`` or
+    ``strategy='dp'`` as a replacement. If you need DDP2, you will need ``torch < 1.9``,
+    ``pytorch-lightning < 1.5``, and set it as ``accelerator='ddp2'``.
```

Comment:
```suggestion
    ``pytorch-lightning < 1.6``, and set it as ``accelerator='ddp2'``.
```
it's present in v1.6 release?

Reply:
It has to be PL < 1.5. The feature is broken since then.

Reply:
oh okay!